src/modules/socketClient/SocketClient.ts

```diff
@@ -0,0 +1,42 @@
+import { io, Socket } from "socket.io-client";
+
+class SocketClient {
+
+  socket: Socket
+  isSocketConnected: boolean
+  
+  constructor() {
+    this.isSocketConnected = false;
+    this.socket = io("http://localhost:5000");
+    
+    this.checkSocketConnection().then(() => {
+      this.isSocketConnected = this.socket.connected
+    })
+  }
+
+  async checkSocketConnection(): Promise<any>{
+    await new Promise((resolve, reject) => {
+      this.socket.on('connect', () => {
+        console.log('Socket Connected? ', this.socket.connected);
```

Comment:
cleanup

Reply:
I need this console.log to check if the connection is made.
I will need to put it back again.

Any better way you can suggest?
Any build, or pre-commit process that would remove the console.logs automatically?

Reply:
Look at `git stash` and `git rebase`